Common/Product/TestAdapter/VisualStudioApp.cs

```diff
@@ -119,7 +118,7 @@ private static DTE GetDTE(int processId)
                         // Do nothing, there is something in the ROT that we do not have access to.
                     }
 
-                    if (!string.IsNullOrEmpty(name) && string.Equals(name, progId, StringComparison.Ordinal))
+                    if (StringComparer.Ordinal.Equals(name, progId))
```

Comment:
Looks slightly off now - should be:
if (!string.IsNullOrEmpty(name) && StringComparer.Ordinal.Equals(name, progId))